src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -91,10 +101,47 @@ def load(self) -> list[Rule]:
             msg = "Please run: databricks labs ucx table-mapping"
             raise ValueError(msg) from None
 
-    def skip_table(self, backend: StatementExecutionBackend, schema: str, table: str):
+    def get_tables_to_migrate(self, tables_crawler: TablesCrawler):
+        upgraded_tables = self._get_upgraded_tables()
+
+        tables_to_validate = []
+        rules = self.load()
+        crawled_tables = tables_crawler.snapshot()
+        crawled_tables_keys = {crawled_table.key: crawled_table for crawled_table in crawled_tables}
+        # Deducing the set of databases in the crawled tables
+        crawled_databases = {crawled_table.database for crawled_table in crawled_tables}
+
+        # Getting all the source tables from the rules
+        source_databases = {rule.src_schema for rule in rules if rule.src_schema in crawled_databases}
+        validated_databases = self._get_validated_databases(source_databases)
```

Comment:
Validated is the wrong word here
```suggestion
        databases_in_scope = self._get_databases_in_scope(source_databases)
```

Reply:
Done